src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -99,7 +99,9 @@ InputStateMachineEngine::InputStateMachineEngine(std::unique_ptr<IInteractDispat
 InputStateMachineEngine::InputStateMachineEngine(std::unique_ptr<IInteractDispatch> pDispatch, const bool lookingForDSR) :
     _pDispatch(std::move(pDispatch)),
     _lookingForDSR(lookingForDSR),
-    _pfnFlushToInputQueue(nullptr)
+    _pfnFlushToInputQueue(nullptr),
+    _doubleClickTime(GetDoubleClickTime()),
+    _lastMouseClickTime(time(nullptr))

```

Comment:
Update: hm this causes our mouse click test to fail because we set the double_click flag (since the test is able to send a left mouse event _immediately_)